packages/flutter/lib/src/cupertino/bottom_tab_bar.dart

```diff
@@ -124,13 +126,18 @@ class CupertinoTabBar extends StatelessWidget implements PreferredSizeWidget {
   /// Must not be null.
   final double iconSize;
 
+  /// The height of the [CupertinoTabBar].
+  ///
+  /// Defaults to 50.0. Must not be null.
```

Comment:
might be good to add some detail here, e.g. saying this is used to set [preferredSize].

Reply:
(doesn't have to be in this PR)